packages/core/story-structure.js

```diff
@@ -140,6 +140,10 @@ const s = [
           'c/TearsheetShell',
         ],
       },
+      {
+        n: 'Novice to pro',
```

Comment:
**Novice to pro** has been designated as a folder directly beneath **IBM Products**